airbyte-integrations/connectors/destination-duckdb/metadata.yaml

```
@@ -14,6 +16,11 @@ data:
     oss:
       enabled: true
   releaseStage: alpha
+  releases:
+    breakingChanges:
+      0.3.0:
```

```suggestion
      1.0.0:
```
Just adding the major version suggestion here as well

---

Hi, @bnchrch 

https://github.com/airbytehq/airbyte/pull/31744/files#r1370836343

From my response on Augustine's comment above:

> We spoke offline. This is community connector and doesn't receive official support.
>
> Also, DuckDB's own database version files are incompatible across minor versions, and the best way to respect this volatility is to keep pre-1.0 connector status until we have a handle on stability overall.
>
> Last - there's a desire to rewrite this in Java before we give official/certified status to the connector.

---

Ah fair enough @aaronsteers !

I was going to dig my heels in on "no major change no breaking change" but after a discussion with @pedroslopez he pointed out weve been doing this same pattern of minor version changes can be breaking if the connector is v0

langchain is the example that came to mind
https://github.com/airbytehq/airbyte/blob/c92837c8d7e7f5f84f63547f3f98ed541fb4defb/airbyte-integrations/connectors/destination-langchain/metadata.yaml

and that seems to line up with the official interpretation of semver when the version is still 0.

**TLDR: LGTM!**